packages/core/src/lib/__tests__/slider.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import userEvent from '@testing-library/user-event';
+import {
+  act,
+  createEvent,
+  fireEvent,
+  render,
+  screen,
+} from '@testing-library/svelte';
+import Slider from '../slider.svelte';
+
+const customMin = -50;
+const customMax = 50;
+const customStart = -20;
+const customEnd = 30;
+const value = 15;
+const step = 25;
+
+export const delay = async (time: number) => {
+  await new Promise((resolve) => {
+    setTimeout(resolve, time);
+  });
+};
```

Comment:
As of Node v18 (which we're on) you can use `node:timers/promises` for this instead: https://nodejs.org/dist/latest-v18.x/docs/api/timers.html#timerspromisessettimeoutdelay-value-options